tdp/core/runner/operation_runner.py

```diff
@@ -110,26 +110,20 @@ def get_deployment_args(self):
     def from_dag(
         dag, targets=None, sources=None, filter_expression=None, restart=False
     ):
-        operation_names = dag.get_operations(sources=sources, targets=targets)
+        operations = dag.get_operations(sources=sources, targets=targets)
         if filter_expression and hasattr(filter_expression, "search"):
-            operation_names = dag.filter_operations_regex(
-                operation_names, filter_expression
-            )
+            operations = dag.filter_operations_regex(operations, filter_expression)
             str_filter = filter_expression.pattern
             filter_type = FilterTypeEnum.REGEX.value
         elif filter_expression:
-            operation_names = dag.filter_operations_glob(
-                operation_names, filter_expression
-            )
+            operations = dag.filter_operations_glob(operations, filter_expression)
             str_filter = filter_expression
             filter_type = FilterTypeEnum.GLOB.value
         else:
             str_filter = None
             filter_type = None
-        # Converting operations names into their actual Operation object
-        operations = itemgetter(*operation_names)(dag.operations)
```

Comment:
Can you remove the import associated with `itemgetter` ?

Reply:
done

Reply:
A linter is missing for checking unused import :)